res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -381,6 +416,21 @@ PioneerDDJSX.init = function(id) {
             PioneerDDJSX.initDeck(index);
         }
     }
+
+    // init effects section:
+    PioneerDDJSX.effectUnit = [];
+    PioneerDDJSX.effectUnit[1] = new PioneerDDJSX.EffectUnit(1);
```

Comment:
Sorry, I copied and pasted that from an old example. The functions were renamed to inSetParameter/inGetParameter to make the API more consistent:
```
PioneerDDJSX.effectUnit[1].dryWetKnob.input = function (channel, control, value, status, group) {
        this.inSetParameter(this.inGetParameter() + PioneerDDJSX.getRotaryDelta(value) / 30);
};
```